resources/monitoring/templates/NOTES.txt

```diff
@@ -1,5 +1,4 @@
-The Prometheus Operator has been installed. Check its status by running:
-  kubectl --namespace {{ $.Release.Namespace }} get pods -l "release={{ $.Release.Name }}"
+{{ $.Chart.Name }} has been installed. Check its status by running:
```

Comment:
We deleted most of the NOTES.txt files as they are not used and yet another something to maintain. Please delete this one as well.

Reply:
Good to know!